btrfs.go

```diff
@@ -365,7 +365,7 @@ func openSubvolDir(path string) (*os.File, error) {
 }
 
 func isStatfsSubvol(statfs *syscall.Statfs_t) error {
-	if statfs.Type != C.BTRFS_SUPER_MAGIC {
+	if int64(statfs.Type) != int64(C.BTRFS_SUPER_MAGIC) {
```

Comment:
If you know off the top of your head, what were the original types on amd64 and on arm?